weldx/schemas/weldx.bam.de/weldx/core/graph/di_node-1.0.0.yaml

```
@@ -32,7 +32,7 @@ examples:
 type: object
 properties:
   name:
-    oneOf:
+    anyOf:
       - type: string
       - tag: "asdf://weldx.bam.de/weldx/tags/uuid-1.0.0"
```

```suggestion
    type: string
```
Only using `type: string` instead of the combiner structure would also work, but I find this more explicit in what types are supported

---

Explicit is better than implicit - Would leave it as is :wink: